src/controls.actions.js

```diff
@@ -231,7 +233,7 @@
         control = target.controls[transform.corner],
         zoom = target.canvas.getZoom(),
         padding = target.padding / zoom,
-        localPoint = target.toLocalPoint(new fabric.Point(x, y), originX, originY);
+        localPoint = target.normalizePoint(new fabric.Point(x, y), originX, originY);
```

Comment:
i need to check if getLocalPoint gets used for other things.

Reply:
I checked (and didn't find, but didn't want to break) and there's another function that is called for a same purpose that is used by one method:
https://github.com/fabricjs/fabric.js/blob/794b6b4a9461b9e79b894784dcfbf4112c90fa7b/src/shapes/object.class.js#L1933

Reply:
> This one

_Originally posted by @ShaMan123 in https://github.com/fabricjs/fabric.js/pull/7599#r785279349_